web/templates/rest-api-doc.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AmpliPi - REST API</title>
+    <!-- needed for adaptive design -->
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="/static/imgs/amplipi-small-logo.png">
+    <link href="/static/css/libs/google-fonts/fonts.css" rel="stylesheet">
+
+    <!--
+    ReDoc doesn't change outer page styles
+    -->
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <script type="module" src="/static/js/libs/rapidoc/9.3.3/rapidoc-min.js"></script>
+    <rapi-doc spec-url="/openapi.yaml" regular-font="Open Sans" mono-font="Roboto Mono" render-style="read" show-header="false" theme="dark"  header-color="#47AFE8" primary-color="#47AFE8" allow-authentication="false">
+      <!-- NOTE: slot="" is used to put inner html where you want it" -->
+      <!-- Image -->
+      <img slot="nav-logo" src="/static/imgs/amplipi_icon_black.png"
+           style="width:90%; display: block; margin-left: auto;margin-right: auto">
+      <!-- content at the top -->
+      <!-- content at the bottom -->
+      <p slot="footer" style="margin:0; padding:16px 36px; background-color:#47AFE8; color:#fff; text-align:center;">
+        © Micronova 2022
```

Comment:
It's not 2022 anymore

Reply:
is there a reason we have to hard code the year like that?

Reply:
I don't know, you added this file but did it come from somewhere else?

Reply:
yeah, it's part of the api docs, they got ripped out by someone at some point